### PR TITLE
Adds Hand Raising

### PR DESCRIPTION
**Pull request by matttheficus**
## What Does This PR Do
Pointing to a slapper in your hand now states:

`**John Doe** raises their hand!`

It still shows the slapper in the little point blurb that goes above your head, for added attention grabbing.

## Why It's Good For The Game
Little fluff things are nice.

## Testing
Pointed at my hand.

## Changelog
:cl:
add: Added the ability to raise your hand.
/:cl:

### CODE REVIEW

**Comment by Furrior** [comment]
Don't like the code, but the idea is nice

**Comment by Christasmurf** [comment]
image

**Comment by Furrior** [comment]
> image
cooltext452578215788095

**Review by Henri215 — APPROVED** [review]

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -286,6 +286,10 @@
 		visible_message("<span class='danger'>[src] points [trick] at- and [trick] goes off in their hand!</span>")
 		trick.shoot_gun(src)
 
+	if(istype(hand_item, /obj/item/slapper))
+		visible_message("<b>[src]</b> raises their hand!")
```
Comment (lewcc) [inline code/modules/mob/living/living.dm:290]:
```suggestion
		visible_message("<b>[src]</b> raises [p_their()] hand!")
```

**Review by lewcc — COMMENTED** [review]
Personally I'm fine with the way this is done, but it's worth looking into a cleaner way to do it.

File: code/modules/mob/living/living.dm [hunk 2]
```diff
@@ -286,6 +286,10 @@
 		visible_message("<span class='danger'>[src] points [trick] at- and [trick] goes off in their hand!</span>")
 		trick.shoot_gun(src)
 
+	if(istype(hand_item, /obj/item/slapper))
```
Comment (Contrabang) [inline code/modules/mob/living/living.dm:289]: Assuming you only want it to happen while the slapper is in your active hand while you point to it, and not just any time you're holding a slapper as your active hand and you point
```suggestion
	if(istype(hand_item, /obj/item/slapper) && hand_item == A)
```

**Review by Contrabang — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/living.dm [hunk 3]
```diff
@@ -268,25 +268,15 @@
...
-	if(istype(hand_item, /obj/item/toy/russian_revolver/trick_revolver) && A != hand_item)
-		var/obj/item/toy/russian_revolver/trick_revolver/trick = hand_item
-		visible_message("<span class='danger'>[src] points [trick] at- and [trick] goes off in their hand!</span>")
-		trick.shoot_gun(src)
+	var/pointed_object = "\the [A]"
+	if(A.loc in src)
+		pointed_object += " inside [A.loc]"
 
-	visible_message("<b>[src]</b> points to [pointed_object]")
+		visible_message("<b>[src]</b> points to [pointed_object]")
```
Comment (Contrabang) [inline code/modules/mob/living/living.dm:279]:
```suggestion
	visible_message("<b>[src]</b> points to [pointed_object]")
```

**Review by Contrabang — APPROVED** [review]
LGTM

**Review by Henri215 — APPROVED** [review]

**Pull request merged**